modules/upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,88 @@
+[server-major-upgrade]
+= Server - Major Upgrade
+
+From time to time, {productname} Server upgrades are labeled as major.
+This is especially the case, if a version upgrade of openSUSE Leap is needed. openSUSE Leap is the underlying base operating system (OS).
+For background information about upgrading openSUSE Leap, see https://doc.opensuse.org/documentation/leap/startup/single-html/book-opensuse-startup/index.html#sec-update-zypper.
+
+Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
+
+[NOTE]
+====
+If the [command]``server-migrator.sh`` script is not already installed on your {productname} Server system, update the [package]``susemanager`` package:
++
+----
+zypper ref
+zypper up susemanager
+----
+====
+
+The [command]``server-migrator.sh`` script will migrate {productname} Server to the latest version.
+This implies upgrading the underlying operating system to version 15.2.
+
+// During migration the services need to be shut down and after successful migration the server needs to be rebooted manually.
```

Comment:
I think this should not be commented, and most probably it should be a warning.

Reply:
ok, it's rephrased and enabled now.